README.md

```diff
@@ -1,150 +1,119 @@
-`https://github.com/ghopper/2015-07-01-esu` and the website for your
-workshop will be `https://ghopper.github.io/2015-07-01-esu`.  (Note:
-the workshop coordinators will want the former URL, not the latter.)
+7.  At this point, you should have a page like this:
 
-## Manual Setup
+    ![](http://software-carpentry.org/img/workshop-template/using-github-import.png)
 
-You can set up your repository manually instead of using the automated
-`create` script.  As above, we will assume that your user ID is
-`ghopper` and the identifier for your workshop is `2015-07-01-esu`.
+    You can now click "Begin Import".
+    When the process is done,
+    you can click "Continue to repository" to visit your newly-created repository.
 
-1.  Create an empty repository on GitHub called `2015-07-01-esu`.
+## Customizing Your Website
 
-2.  Clone the template repository to your computer in a directory with
-    the same name as your workshop identifier:
-
-    ~~~
-    $ git clone -b gh-pages -o upstream https://github.com/swcarpentry/workshop-template.git 2015-07-01-esu
-    ~~~
-
-3.  Go into that directory using
-
-    ~~~
-    $ cd 2015-07-01-esu
-    ~~~
+1.  Go into your newly-created repository,
+    which will be at `http://github.com/your_username/YYYY-MM-DD-site`.
+    For example,
+    if `your_username` is `gvwilson`,
+    the repository's URL will be `https://github.com/gvwilson/2015-07-01-mistaktonic`.
 
-4.  Add your GitHub repository as a remote called `origin` using
+2.  Edit `index.html`.
+    You can do this in the browser by clicking on it in the file view
+    and then selecting the pencil icon in the menu bar:
 
-    ~~~
-    $ git remote add origin https://github.com/ghopper/2015-07-01-esu.git
-    ~~~
+    ![](http://software-carpentry.org/img/workshop-template/edit-index-file-menu-bar.png)
 
-5.  Edit `index.html`.  Hints are embedded in the file, and full
-    instructions are in [CUSTOMIZATION.md](CUSTOMIZATION.md).
+    or you can clone the repository to your desktop,
+    edit `index.html` there,
+    and push your changes back to the repository.
 
-6.  If you have installed the software described below:
+3.  When you are done editing,
+    you can preview your website at its GitHub Pages URL,
+    which is `http://your_id.github.com/YYYY-MM-DD-site`.
+    For example,
+    the URL for the website we have just created is
+    `http://gvwilson.github.io/2015-07-01-miskatonic`.
 
-    1.  Check your changes by running `tools/check.py` inside your
-        repository.
+4.  Edit `_config.yml` in the same way
+    so that `lesson_repo` and `lesson_site`
+    are the URLs of your repository (e.g., `https://github.com/gvwilson/2015-07-01-mistaktonic`)
+    and your GitHub Pages website (e.g., `http://gvwilson.github.io/2015-07-01-miskatonic`).
 
-    2.  Preview your changes by running `tools/preview` and looking at
-        `_site/index.html`.
+Editing hints are embedded in `index.html`,
```

Comment:
This note would be more helpful if placed sooner. Perhaps at the end of the instruction step on editing index.html.